VideoPlusChat/src/main/java/com/example/videochatcomposesample/ui/views/ChatPage.kt

```diff
@@ -0,0 +1,185 @@
+package com.example.videochatcomposesample.ui.views
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.BottomAppBar
+import androidx.compose.material.Icon
+import androidx.compose.material.LocalElevationOverlay
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowForward
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.focus.FocusManager
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.videochatcomposesample.ChatMessage
+import com.example.videochatcomposesample.models.ChatModel
+
+var chatModel: ChatModel = ChatModel()
+
+@Composable
+fun ChatPage(focusManager: FocusManager) {
+    Scaffold(
+        topBar = { ChatTitle() },
+        content = { Chat() },
+        bottomBar = { ChatInputBar(focusManager = focusManager) }
+    )
+}
+
+@Composable
+fun Chat() {
+
+    Box(
+        modifier = Modifier
+            .fillMaxHeight(0.85f)
+            .fillMaxWidth()
+            .background(MaterialTheme.colors.secondary)
+    ) {
```

Comment:
It looks like you can remove the Box and just move these modifier attributes directly to the LazyColumn in `ChatList` - remember that order matters with modifiers so you may have to experiment a little to get the desired result!